source/CommonJobs/CommonJobs.Application/ApplicantFlow/GetEventTypeSuggestionsQuery.cs

```diff
@@ -23,42 +24,58 @@ public GetEventTypeSuggestionsQuery(string term, int maxSuggestions = 8)
+            //    .OrderByDescending(x => x.Predefined).ThenBy(x => x.Text)
+            //    .Select(x => x.Text)
+            //    .As<string>()
+            //    .Distinct()
+            //    .Take(MaxSuggestions * 2); //Padding porque no puedo filtrar los vacíos antes
+
+            //var results = query.AsEnumerable()
+            //    .Where(x => !string.IsNullOrWhiteSpace(x))
+            //    .Take(MaxSuggestions).ToList();
 
-            if (results.Count < MaxSuggestions)
-            {
-                var suggestionTerms = query.Suggest().Suggestions.Where(x => x != Term).ToArray();
-                if (suggestionTerms.Length > 0)
-                {
-                    var extraQuery = RavenSession.Query<ApplicantEventType_Suggestions.Projection, ApplicantEventType_Suggestions>()
-                        .Search(x => x.Text, string.Join(" ", suggestionTerms))
-                        .OrderByDescending(x => x.Predefined).ThenBy(x => x.Text)
-                        .Select(x => x.Text)
-                        .As<string>()
-                        .Distinct()
-                        .Take((MaxSuggestions - results.Count) * 2); //Padding porque no puedo filtrar los vacíos y duplicados antes
+            //if (results.Count < MaxSuggestions)
+            //{
+            //    var suggestionTerms = query.Suggest().Suggestions.Where(x => x != Term).ToArray();
+            //    if (suggestionTerms.Length > 0)
+            //    {
+            //        var extraQuery = RavenSession.Query<ApplicantEventType_Suggestions.Projection, ApplicantEventType_Suggestions>()
+            //            .Search(x => x.Text, string.Join(" ", suggestionTerms))
+            //            .OrderByDescending(x => x.Predefined).ThenBy(x => x.Text)
+            //            .Select(x => x.Text)
+            //            .As<string>()
+            //            .Distinct()
+            //            .Take((MaxSuggestions - results.Count) * 2); //Padding porque no puedo filtrar los vacíos y duplicados antes
 
-                    var extraResults = extraQuery.ToList();
+            //        var extraResults = extraQuery.ToList();
 
-                    extraResults = extraResults
-                        .Where(x => !string.IsNullOrWhiteSpace(x))
-                        .Where(x => !results.Contains(x))
-                        .Take(MaxSuggestions - results.Count)
-                        .ToList();
+            //        extraResults = extraResults
+            //            .Where(x => !string.IsNullOrWhiteSpace(x))
+            //            .Where(x => !results.Contains(x))
+            //            .Take(MaxSuggestions - results.Count)
+            //            .ToList();
 
-                    results.AddRange(extraResults);
-                }
-            }
+            //        results.AddRange(extraResults);
+            //    }
+            //}
```

Comment:
This is partial. The old query wasn't working with the new API so I had to rework it, but it's not good enough. This also happens with other "suggestions queries"